pom.xml

```diff
@@ -30,12 +30,25 @@
     </modules>
 
     <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+        <maven.min.version>3.5.0</maven.min.version>
+
+        <version.net.revelc.code.formatter-maven-plugin>2.11.0</version.net.revelc.code.formatter-maven-plugin>
```

Comment:
there is a newer version https://mvnrepository.com/artifact/net.revelc.code.formatter/formatter-maven-plugin

Reply:
I know, do we want to update really?

Reply:
this is done in https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/269
maybe we can use both PRs after all

Reply:
updated